### PR TITLE
Mixed content because of secure https protocol

### PR DESCRIPTION
**Pull request by shivamsngh**
Mixed content because of secure https protocol is stopping the scripts and css to load and render.

### CODE REVIEW

**Comment by aaronsofaly** [comment]
Nice work @shivamsngh!

We should probably update all asset references to `{{site.github.url}}`.

- https://github.com/chaijs/chaijs.github.io/blob/master/_includes/head.html#L9
- https://github.com/chaijs/chaijs.github.io/blob/master/_includes/head.html#L13
- https://github.com/chaijs/chaijs.github.io/blob/master/_includes/head.html#L14 (done)
- https://github.com/chaijs/chaijs.github.io/blob/master/_includes/head.html#L15 (done)
- https://github.com/chaijs/chaijs.github.io/blob/master/_includes/head.html#L18 (done)
- https://github.com/chaijs/chaijs.github.io/blob/master/_includes/head.html#L19 (done)
- https://github.com/chaijs/chaijs.github.io/blob/master/index.html#L147
- https://github.com/chaijs/chaijs.github.io/blob/master/plugins/index.html#L33
- https://github.com/chaijs/chaijs.github.io/blob/master/_layouts/test.html#L16
- https://github.com/chaijs/chaijs.github.io/blob/master/_layouts/guide.html#L43

@keithamus, is it necessary to keep using `{{site.github.url}}` instead of just using a relative path? For example, using `<script src="/js/main.js"></script>` instead of using `<script src="{{ "/js/main.js" | prepend: site.github.url | replace: 'http://', '//' }}"></script>`?

**Comment by keithamus** [comment]
I'm happy to use absolute paths. @shivamsngh are you happy to make those changes?

**Comment by shivamsngh** [comment]
> Sure, I will make those chages and create the pull request

@keithamus  @aaronsofaly  I have pushed the changes to original pull request. Please review and merge.

File: _includes/apiSidebar.html [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-    <a href="{{ site.github.url }}/api/assert/" class="item{% if page.url == '/api/assert/' %} active{% endif %}">
+    <a href="https://www.chaijs.com/api/assert/" class="item{% if page.url == '/api/assert/' %} active{% endif %}">
         Assert
     </a>
-    <a href="{{ site.github.url }}/api/plugins/" class="item{% if page.url == '/api/plugins/' %} active{% endif %}">
+    <a href="https://www.chaijs.com/api/plugins/" class="item{% if page.url == '/api/plugins/' %} active{% endif %}">
         Plugin Utilities
     </a>
-    <a href="{{ site.github.url }}/api/test/" class="item{% if page.url == '/api/test/' %} active{% endif %}">
+    <a href="https://www.chaijs.com/api/test/" class="item{% if page.url == '/api/test/' %} active{% endif %}">
```
[the remark this answers is not in the document]
Comment (shivamsngh) [reply _includes/apiSidebar.html]: Converted absolute URLs to relative URLs. Also changed some of http resources to https (like google groups). 
"make" builds and starts successfully.

**Comment by keithamus** [comment]
thanks for working on this @shivamsngh! Thanks for the assist @aaronsofaly

**Pull request merged**